app/scripts/components/boilerplate/dashboard-btn.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+require('stylesheets/boilerplate/buttons');
+
+const DashboardBtn = React.createClass({
+  render() {
+    return (
+      <div>
+      <a className="button-link" href="/waterpoints/" >water points</a>
```

Comment:
Hey @uniphil it would be awesome to chat about the last point. I replaced <a> with <Link>  otherwise. I'll ping you tomorrow when I'm back.